contrib/datawave-quickstart/bin/services/datawave/test-web/tests/QueryMetrics.test

```diff
@@ -13,7 +13,7 @@ setCurlData \
         begin=19701231 \
         end=20500101 \
         pagesize=1 \
-        auths=PUBLIC \
+        auths=PRIVATE \
```

Comment:
This is a curious change required for Java 11... why is this needed?

Reply:
Found that the test currently fails without the PRIVATE auth, technically a separate issue. Will address in a different PR